Sources/CloudEnvironment/CloudFunctionsCredentials.swift

```diff
@@ -16,21 +16,22 @@
 
 import Foundation
 
-/// CloudFunctionsCredentials class
+/// Contains the credentials for a Cloud Functions instance. You will typically
+/// receive an instance of this type through `cloudEnv.getCloudFunctionsCredentials(name: String)`.
 ///
-/// Contains the credentials for a Cloud Functions instance.
+/// Reference [Cloud Functions](https://cloud.ibm.com/docs/openwhisk?topic=cloud-functions-index#getting-started-with-cloud-functions).
```

Comment:
Here we link to the docs, but given that most other places link to the catalog services, we should link to https://cloud.ibm.com/openwhisk

Reply:
I've implemented this in the latest commit.